### PR TITLE
Add meeting minutes for SVSM Development Call on September 4th, 2024

### PR DESCRIPTION
**Pull request by joergroedel**

### CODE REVIEW

File: Meetings/devel-call-2024-09-04.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Meeting Minutes: SVSM Development Call (September 4th, 2024)
+
+## Attendees:
+
+Adam Dunlap, Diego Gonzalez Villalobos, Dionna Glaze, Hajira Zaman, Huibo Wang, James Bottomley, Joerg Roedel, Nicolai Stange, Oliver Steffen, Peter Fang, Rama AMD, Roy Hopkins, Stefano Garzarella, Tom Dohrmann, Vijay Dhanraj
+
+## Topics:
+
+### Early Attestation Process Document
+
```
Comment (stefano-garzarella) [inline Meetings/devel-call-2024-09-04.md:10]: What about add a link to the document?

https://docs.google.com/document/d/11ZsxP8jsviP3ddp9Hrn0rf6inttNw_Pbnz0psXlxlPs

File: Meetings/devel-call-2024-09-04.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Provisioning: The initial state creation should be done offline using a tool that can register the secret into the KBS.
+* TPM State: The TPM state should be considered as a secret that needs to be stored and recovered on subsequent boots.
+* KBS as Untrusted Entity: The KBS should be treated as a read-only, untrusted entity.
+
+#### Communication Channel and Protocol
+
+* HTTP Proxy: The group discussed using an HTTP proxy as a simple communication channel between SVSM and the KBS.
+* Attestation Protocol: The attestation protocol should be implemented in the SVSM.
+* Negotiation Phase: A negotiation phase between SVSM and the KBS is needed to establish communication parameters.
+* Attested TLS: Using attested TLS for communication between SVSM and the KBS is recommended.
```
Comment (stefano-garzarella) [inline Meetings/devel-call-2024-09-04.md:21]: IIUC @deeglaze suggested `aTLS` between the proxy and SVSM if we implement our protocol between the two, and allow the attestation protocol implementation into the proxy.

So aTLS can be used to negotiate hash and crypto algorithms between SVSM and the proxy.
See https://docs.google.com/document/d/11ZsxP8jsviP3ddp9Hrn0rf6inttNw_Pbnz0psXlxlPs/edit?disco=AAABRuYNYHo

@deeglaze please correct me if I'm wrong.

File: Meetings/devel-call-2024-09-04.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+#### Persistent State
+
+* Challenges: The group discussed the challenges of maintaining a persistent state in the TPM.
+* Security Concerns: Persistent state is vulnerable to replay and cloning attacks
```
Comment (stefano-garzarella) [inline Meetings/devel-call-2024-09-04.md:26]:
```suggestion
* Security Concerns: Persistent state is vulnerable to rollback and cloning attacks
```

File: Meetings/devel-call-2024-09-04.md (hunk 2 above)
Comment (stefano-garzarella) [reply Meetings/devel-call-2024-09-04.md:21]: Also in this case, maybe we can add a link to the aTLS proposal: https://datatracker.ietf.org/doc/draft-fossati-tls-attestation/

Comment (deeglaze) [reply Meetings/devel-call-2024-09-04.md:21]: I don't know what crypto algorithms you're talking about. I'm thinking we'd just have a transport layer secure channel terminated in the proxy. The negotiation with whatever services can be GET requests for REST or something, but every service is going to have its own quirks. We ought not require that the VMM itself constructs the application requests to the connected server.

File: Meetings/devel-call-2024-09-04.md (hunk 3 above)
Comment (deeglaze) [reply Meetings/devel-call-2024-09-04.md:26]: Replay and rollback are synonymous, no?

File: Meetings/devel-call-2024-09-04.md (hunk 2 above)
Comment (stefano-garzarella) [reply Meetings/devel-call-2024-09-04.md:21]: > I'm thinking we'd just have a transport layer secure channel terminated in the proxy

And started from where?

Comment (deeglaze) [reply Meetings/devel-call-2024-09-04.md:21]: Let's move this discussion to the early attestation doc.

File: Meetings/devel-call-2024-09-04.md (hunk 1 above)
Comment (joergroedel) [reply Meetings/devel-call-2024-09-04.md:10]: Link is added.

File: Meetings/devel-call-2024-09-04.md (hunk 2 above)
Comment (joergroedel) [reply Meetings/devel-call-2024-09-04.md:22]: Discussion moved to doc. Resolving.

**Pull request merged**